homeassistant/components/downloader/services.py

```diff
@@ -151,9 +175,14 @@ def async_setup_services(hass: HomeAssistant) -> None:
         download_file,
         schema=vol.Schema(
             {
+                vol.Optional(ATTR_AUTH_TYPE, default=HTTP_BASIC_AUTHENTICATION): vol.In(
```

Comment:
I don't think you should have a default here.
```suggestion
                vol.Optional(CONF_AUTHENTICATION): vol.In(
```